cfgov/cal/admin.py

```diff
@@ -51,28 +57,31 @@ def changelist_view(self, request, extra_context=None):
             if icsfile:
                 file_type = icsfile.content_type.split('/')[0]
 
-                if(file_type != "text"):
-                    error.append( (['There was an error with the File'],["Incorrect File Type"]) )
-                    context.update({ 'error': error,})
-                    return super(CFPBCalendarImportICSFileAdmin,self).changelist_view(request,context)
+                if (file_type != "text"):
```

Comment:
Not PEP8, and maybe out of scope for this PR, but it'd be nice to standardize on use of single quotes and change double quotes where we see them. See the line that starts with `error.append` that adds two strings that use different quotes!